### DISCUSSION TITLE
Getting Started with reTerminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#216] [Q&A]
# Getting Started with reTerminal | Seeed Studio Wiki

Getting Started with reTerminal

https://wiki.seeedstudio.com/reTerminal/

**Comment by CHARACODES** [comment]
after update os RTC not working how to slove that

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by CHARACODES** [comment]
If we enable 1-wire RTC not work how to slove that

**Reply by xzxcessarr** [reply]
Hello, From your discription we do not acquire the detail problem about your device, could you tell more about it? 
1. Which firmware is you using now 
2. Which function you used to distinguish the whether the RTC is work well?
3. Is the battery well?
Thanks!

**Comment by FraserElectronics** [comment]
This unit is JUNK.
First power up and it does updates and bricks itself.
Follow the steps above to flash a new image and it does not even see the unit as a mass storage device on three different PC's with three different USB-C leads. What a joke ! I have returned the unit for a full refund. Buyers beware !!!!

**Reply by jaredtwomey** [reply]
I know this is an older comment, but just so folks aren't as confused.  When you first set it up and if you do the updates, the device loses the drivers it needs.  It's not bricked, you just need to plug it into a monitor, or SSH into it over the network to reinstall the drivers as explained in this tutorial

**Reply by gswagg** [reply]
Adding to this, in order to further clarify, you must also download and run "rpiboot" in order to get your PC to recognize it as a mass storage device, also explained in this tutorial.

**Comment by CHARACODES** [comment]
How To control WS2816C LEDs reTerminal.
